dna/data.py

```diff
@@ -78,14 +87,14 @@ def group_json_objects(json_objects, group_key):
     grouped_objects = {}
     for i, obj in enumerate(json_objects):
         group = obj[group_key]
+        group = tuple(group)
```

Comment:
why is this needed?

Reply:
I think I was trying to hash with a list -> if you convert a set to an immutable type it lets you use it as a key.  I think we decided to go another route though so we can remove it.